applications/mne_analyze/libs/anShared/Model/fiffrawviewmodel.h

```diff
@@ -911,7 +911,7 @@ class AnnotationData {
 private:
 
     int m_iSample;
-    int m_iEventType;
+    //int m_iEventType;
```

Comment:
Is this member used anywhere? If not, we should consider removing it completley.

Reply:
I don't want to undo anybody's work. just want the <<noise>> to go away 😅